src/components/pill/pill.pw.tsx

```diff
@@ -366,6 +468,32 @@ test.describe("should check for Accessibility tests", () => {
     });
   });
 
+  ([
+    "warning",
+    "neutral",
+    "negative",
+    "positive",
+    "information",
+    "neutralWhite",
+  ] as const).forEach((color) => {
+    test(`should render colorVariant with fill as ${color} when isDarkBackground is set to true for accessibility `, async ({
+      mount,
+      page,
+    }) => {
+      await mount(
+        <PillOnDarkBb
+          pillRole="status"
+          colorVariant={color}
+          fill
+          isDarkBackground
+        />
+      );
+
+      // TODO: color contrast failing on information variant with dark bg and fill, DS is aware
```

Comment:
praise: good spot and initiative here

It might be worth creating a ticket so it can be tracked and adding the ticket number to this comment. Not ideal that we have another set of failing colour contrast tests. Hopefully, it will be resolved soon